src/platform_impl/macos/window.rs

```diff
@@ -272,6 +274,10 @@ fn create_window(
         let _: () = msg_send![parent as id, addChildWindow: *ns_window ordered: NSWindowOrderingMode::NSWindowAbove];
       }
 
+      if !pl_attrs.automatic_tabbing {
+        NSWindow::setAllowsAutomaticWindowTabbing_(*ns_window, NO);
```

Comment:
```suggestion
        ns_window.setAllowsAutomaticWindowTabbing_(NO);
```

Reply:
This doesn't work, `setAllowsAutomaticWindowTabbing_` only works when called as a static method, not an instance method.

Reply:
yeah I was confused when I reviewed the PR, didn't think that it would require `Self` which seems like it might be a bug in `cocoa` but shouldn't concern this PR.

Reply:
Yes I also thought it was a strange API, not sure why it wants `Self`since it doesn't use it...